Update landing page docs. (#29)
vercel/edge-runtime: docs/pages/index.mdx

---
title: What is Edge Runtime
description: The Edge Runtime is a toolkit for developing, testing, and defining the runtime Web APIs for Edge infrastructure.
---

import Features from '../components/features'

# The Edge Runtime

The Edge Runtime is designed to help framework authors adopt edge computing and provide open-source tooling built on Web standards. It’s designed to be integrated into frameworks (like Next.js) and not for usage in application code. This runtime is purposefully minimal and designed for security and speed.

> When your framework is built on the Edge Runtime, it runs everywhere Node.js does. When deployed to infrastructure like Vercel, frameworks like Next.js can automatically provision the optimal infrastructure for you: either at the edge or at origin.

Author: ```suggestion
When your framework is built on the Edge Runtime, it runs everywhere Node.js does. When deployed to infrastructure like Vercel, frameworks like Next.js can automatically provision the optimal infrastructure for you.
```

> The Edge Runtime is a subset of standard Web APIs, giving you compatability and interoperability with the highest number of environments. This project is developed as part of [WinterCG](https://wintercg.org) – a community group between Vercel, Cloudflare, Deno, Shopify, and more.

Author: ```suggestion
The Edge Runtime is a subset of standard Web APIs, giving you compatibility and interoperability with the highest number of environments. This project is developed as part of [WinterCG](https://wintercg.org) – a community group between Vercel, Cloudflare, Deno, Shopify, and more. The term “Edge” refers to the orientation toward instant serverless compute environments and not a specific set of locations.
```

<Features />

## Using the Edge Runtime Locally

> When developing and testing locally, the Edge RUntime will polyfill Web APIs and ensure compatibility with the Node.js layer.

Reviewer: ```suggestion
When developing and testing locally, the Edge Runtime will polyfill Web APIs and ensure compatibility with the Node.js layer.
```
